R/merge_regimen.R

```diff
@@ -9,6 +9,7 @@
 #' @return Merged regimens
 merge_regimen <- function(regimens) {
   cols <- c("interval", "n", "type", "t_inf", "dose_times", "dose_amts", "first_dose_time")
+  if("cmt" %in% names(regimens[[1]])) { cols <- c(cols, "cmt") }
```

Comment:
only merge `cmt` values if present. This is a bit ugly to have to include. But the way we implemented is that the regimen is in principle agnostic about the model, so we are allowed to define regimens without explicitly stating the `cmt` it will go in, and then rely on defaults or the `cmt_mapping` variable. However, in the case of double absorption, we have to be explicit about the compartments.